app/views/machines/show.html.erb

```diff
@@ -1 +1,12 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+
+<section class="snacks-index">
+  <% @machine.snacks.each do |snack| %>
+    <p><%= snack.name %></p>
+    <p><%= snack.price %></p>
```

Comment:
for displaying price, or even average price down the line, you can look into using number_to_currency to help with formatting.